src/cdp/transport.ts

```diff
@@ -108,10 +109,17 @@ export class WebSocketTransport implements ITransport {
   onmessage?: (message: string, receivedTime: bigint) => void;
   onend?: () => void;
 
-  static create(url: string, cancellationToken: CancellationToken): Promise<WebSocketTransport> {
+  static async create(
+    url: string,
+    cancellationToken: CancellationToken,
+  ): Promise<WebSocketTransport> {
+    const isSecure = !url.startsWith('ws://');
```

Comment:
any reason this couldn't be
```ts
const isSecure = url.startsWith('wss://')
```

Reply:
I prefer to treat things as secure unless explicitly specified otherwise. In the current use it doesn't matter that much, but if the code around it changes it might.